appformer-provisioning/appformer-provisioning-backend/src/test/java/org/kie/appformer/provisioning/tests/RestPipelineImplTest.java

```diff
@@ -105,119 +100,152 @@
+                                 put("wildfly-user",
+                                     "admin");
+                                 put("wildfly-password",
+                                     "Admin#70365");
+                                 put("host",
+                                     ip);
+                                 put("port",
+                                     "8080");
+                                 put("management-port",
+                                     "9990");
+                             }
+                         },
+                         pipe,
+                         System.out::println);
 
-        List<org.guvnor.ala.runtime.Runtime> allRuntimes = runtimeRegistry.getRuntimes( 0, 10, "", true );
+        List<org.guvnor.ala.runtime.Runtime> allRuntimes = runtimeRegistry.getRuntimes(0,
+                                                                                       10,
+                                                                                       "",
+                                                                                       true);
 
-        assertEquals( 1, allRuntimes.size() );
+        assertEquals(1,
+                     allRuntimes.size());
 
-        org.guvnor.ala.runtime.Runtime runtime = allRuntimes.get( 0 );
+        org.guvnor.ala.runtime.Runtime runtime = allRuntimes.get(0);
 
-        assertTrue( runtime instanceof WildflyRuntime );
+        assertTrue(runtime instanceof WildflyRuntime);
 
-        WildflyRuntime wildflyRuntime = ( WildflyRuntime ) runtime;
+        WildflyRuntime wildflyRuntime = (WildflyRuntime) runtime;
 
-        final WildflyRuntimeManager runtimeManager = new WildflyRuntimeManager( runtimeRegistry, wildflyAccessInterface );
+        final WildflyRuntimeManager runtimeManager = new WildflyRuntimeManager(runtimeRegistry,
+                                                                               wildflyAccessInterface);
 
-        runtimeManager.start( wildflyRuntime );
+        runtimeManager.start(wildflyRuntime);
 
-        allRuntimes = runtimeRegistry.getRuntimes( 0, 10, "", true );
+        allRuntimes = runtimeRegistry.getRuntimes(0,
+                                                  10,
+                                                  "",
+                                                  true);
 
-        assertEquals( 1, allRuntimes.size() );
+        assertEquals(1,
+                     allRuntimes.size());
 
-        runtime = allRuntimes.get( 0 );
+        runtime = allRuntimes.get(0);
 
-        assertTrue( runtime instanceof WildflyRuntime );
+        assertTrue(runtime instanceof WildflyRuntime);
 
-        wildflyRuntime = ( WildflyRuntime ) runtime;
+        wildflyRuntime = (WildflyRuntime) runtime;
 
-        assertEquals( "Running", wildflyRuntime.getState().getState() );
-        runtimeManager.stop( wildflyRuntime );
+        assertEquals("Running",
```

Comment:
Here too, please use the constant org.guvnor.ala.runtime.RuntimeState.RUNNING
I'm sorry, but the thing is that the sate will return "RUNNING" and not "Running" any more